stan/math/prim/mat/fun/Phi.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef STAN_MATH_PRIM_MAT_FUN_PHI_HPP
+#define STAN_MATH_PRIM_MAT_FUN_PHI_HPP
+
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
+#include <stan/math/prim/scal/fun/Phi.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * Structure to wrap Phi() so it can be vectorized.
+     * @param x Argument variable.
+     * @tparam T Argument type.
+     * @return Unit normal CDF of x.
+     */
+    struct Phi_fun {
+      template <typename T>
+      static inline T fun(const T& x) {
+        using stan::math::Phi;
+        return Phi(x);
+      }
+    };
+
+    /**
+     * Vectorized version of Phi().
+     * @param x Container.
```

Comment:
(fix later) `@tparam` first, with blank line between last line of text and first @ thing.